doc/starterkit/k4MarlinWrapperCLIC/Readme.md

```diff
@@ -154,10 +153,14 @@ k4run clicRec_e4h_input.py --EventDataSvc.input ttbar_edm4hep.root
 
 ### DD4hep Geometry Information
 
-The ``MarlinDD4hep::InitializeDD4hep`` processor can be replaced by the ``k4SimGeant4::GeoSvc`` and the
-``TrackingCellIDEncodingSvc`` the latter of which is part of the k4MarlinWrapper repository.
+The `MarlinDD4hep::InitializeDD4hep` processor can be replaced by the `k4SimGeant4::GeoSvc` and the
+`TrackingCellIDEncodingSvc` the latter of which is part of the k4MarlinWrapper repository.
 This requires removing the wrapped `InitDD4hep` processor from the `algList` and the two new processed be appended to the `ExtSvc` argument in the `ApplicationMgr`.
 
+```{note}
+If you are using a version of `k4MarlinWrapper` that is newer than `v00-11` then this replacement will be done automatically by the `convertMarlinSteeringToGaudi.py` script.
```

Comment:
```suggestion
If you are using a version of `k4MarlinWrapper` that is newer than `v00-11` (February 2025) then this replacement will be done automatically by the `convertMarlinSteeringToGaudi.py` script.
```
I'm not sure if a date is the best solution but something so the average user can tell whether that version is old or not, even if the version in the stack may be different. Long term one can decide it's old.

Reply:
Should we put a dedicated release tag there? That would also give similar information.

Reply:
But there isn't a release after February :confused:

Reply:
Yes, I was refering to that. I have also put February 2025 there, because there is no release yet, for which this would work. It's a bit of a chicken & egg problem here :sweat_smile:

(also it should be March by now :joy:)

Reply:
Added it as April 2025